src/test/java/com/systelab/seed/patient/PatientResourceTest.java

```diff
@@ -194,7 +232,7 @@ public void testDeletePatient() {
                 .then().assertThat().statusCode(200)
                 .extract().as(Patient.class);
 
-        Assertions.assertNotNull(patientCreated);
+        Assertions.assertNotNull(patientCreated, "Patient not created");
```

Comment:
Do you mean "Patient created"? If it is not created, you can not delete it and test the get by non-existing id...

Reply:
This mesage only is displayed when the assertion has failed, so the patient has not been created.

Reply:
I'm sorry, I didn't catch that the string is just if the assert fails, so when patient is not created.

Reply:
"Patient not created" is the message that will be displayed in case the assertion is not null